app/models/comment.rb

```diff
@@ -1,4 +1,4 @@
 class Comment < ApplicationRecord
-  belongs_to :users
-  belongs_to :posts
+  belongs_to :user
+  belongs_to :post
```

Comment:
- [ ] It will be nice to add a method that updates the comments counter for a post. A method similar to this:

```ruby
# app/models/comment.rb
...

def update_comments_counter
  post.increment!(:comments_counter)
end
```